dashboard/lib/services/csta_enrollment.rb

```diff
@@ -51,7 +53,9 @@ def self.submit(first_name:, last_name:, email:, school_district_name:,
         "submission[17_city]" => city.titleize,
         "submission[17_state]" => get_us_state_abbr(state, true),
         "submission[17_zip]" => zip,
-        "submission[19]" => "Yes, I provide my consent."
+        "submission[19]" => "Yes, I provide my consent.",
+        "submission[20]" => professional_role,
```

Comment:
Just confirming: if the dcdo flag is off, these will be `nil` and the submission will be like we didn't even try to set these key-values right?

Reply:
That's the idea! If my ruby is correct, this should be handled in `amazon_future_engineer_controller`, we call to this function with ```professional_role: afe_params['primaryProfessionalRole'] || '',
        grades_teaching: afe_params['gradesTeaching'] || '',```.
        
        So if there is no input there, we'll just pass along "".